python/setup.py

```diff
@@ -54,6 +54,77 @@ def run(self):
         )
 
 
+class Release(Command):
+    description = "Tag and release image"
+    user_options = []
+
+    gs_image = "registry.cn-hongkong.aliyuncs.com/graphscope/graphscope"
+    gie_manager_image = (
+        "registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager"
+    )
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        from graphscope._version import __version__
+        from graphscope._version import git_info
+
+        tag = __version__
+        commit_hash, commit_ref = git_info()
```

Comment:
As there are many plain `return` in `git_info`, which means if there are any errors in `git_info`, you will got a `None` as return value, which will trigger a `TypeError: cannot unpack non-iterable NoneType object` in `commit_hash, commit_ref = git_info()`.